wmfdata/spark.py

```diff
@@ -1,15 +1,22 @@
 from threading import Timer
 import warnings
+import os
 
 import findspark
-findspark.init("/usr/lib/spark2")
-from pyspark.sql import SparkSession
 
+from wmfdata import conda
```

Comment:
The top of init.py says that modules can depend only depend on utils to prevent circular import issues. Can you update that to say depending on conda is okay too?

Also, you may want to import conda there too so it's available after `import wmfdata` like all the other modules. But I doubt it's going to be used by any end users, so perhaps we should just leave it out? Up to you.

Reply:
ok. Lemme know if I did that right...